product_docs/docs/pgd/5/durability/group-commit.mdx

```diff
@@ -108,5 +113,47 @@ transaction abort is requested. If the transaction is already
 decided to be committed at the time the abort request is sent, the transaction
 does eventually COMMIT even though the client might receive an abort message.
 
+See also [Limitations](limitations). 
+
+### Transaction Reconciliation
+
+A group commit transaction’s commit on the origin node is implicitly converted 
+into a two step commit. 
+
+In the first step the transaction is prepared locally and made ready to commit.
+The data is made durable but uncomitted at this stage so other transactions
+cannot see the changes made by this transaction. This prepared transaction gets
+replicated to all remaining nodes via async replication. 
+
+The origin node seeks confirmations from other nodes, as per rules in the group
+commit grammar. If it gets confirmations from minimum required nodes in the
+cluster, it decides to commit this transaction and also send this decision via
+replication to other nodes which will also eventually commit on getting this
+message. 
+
+There is a possibility of failure at various stages. For example, the origin
+node may crash after preparing the transaction. Or the origin and one or more
+replicas may crash. 
+
+This leaves the prepared transactions in the system. The `pg_prepared_xacts`
+view in Postgres can show prepared transactions on a system. The prepared
+transactions could be holding locks and other resources and they therefore need
+to be either aborted or committed. That decision has to be made with a consensus
+of nodes. 
+
+When commit_decision is raft then, raft acts as the reconcilator and these
+transactions are eventually, automatically, reconciled. 
+
+But group commit transactions do not use raft and instead the write lead in the
```

Comment:
This is the difference between a commit_decision of raft and of group (the default).

Reply:
Now reads...

When the commit_decision is `group` then, transactions do not use raft and instead 
the write lead in the cluster performs the role of reconciliator. This is because 
it is the node that is most ahead with respect to changes in its sub-group.